### PR TITLE
Fix crash on Game::checkImbuements

### PR DESCRIPTION
**Pull request by beats-dh**
Crash was occurring because the item was released and then used
 - [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by gerotib** [comment]
Doesnt crash but I think that `CONDITION_INFIGHT` check is missing, it decreases all the time now

**Review by gerotib — APPROVED** [review]
seems fine now, removes only 1 imbuing and doesnt crash

**Comment by dudantas** [comment]
> seems fine now, removes only 1 imbuing and doesnt crash

Thanks for the feedback, let's wait for the review from other members.

**Review by Bipbop69 — APPROVED** [review]

**Review by dudantas — APPROVED** [review]

**Pull request merged**
